src/pages/article/Article.jsx

```diff
@@ -1,13 +1,20 @@
 import { useEffect, useState } from "react";
-import { useParams } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
 import { fetchArticlesById } from "../../api";
+import "./Article.css"
 import moment from "moment";
 
 const Article = () => {
   const [article, setArticle] = useState({});
   const { article_id } = useParams();
   const [isLoading, setIsLoading] = useState(true);
 
+  const navigate = useNavigate();
+
+  const handleClick = (article) => {
+    navigate(`/articles/${article.article_id}/comments`);
```

Comment:
This is an interesting approach, but I think it is quite confusing for the user, and for other devs reading your code. 

It would be a more "standard" user experience to see the comments on the same page as the article, ie not changing the path, or navigating elsewhere. 

If you'd like to stick with an approach that allows users to reveal the comments when they click a button think about whether you could hold a value in state that determines whether the comments are shown or not...?